src/supermarket/spec/models/curry/repository_spec.rb

```diff
@@ -22,4 +22,22 @@
     expect(repository.owner).to eql('chef')
     expect(repository.name).to eql('supermarket')
   end
+
+  describe '#subscribe!' do
+    let(:expected_fqdn) { 'bunniesareadorable' }
+
+    before do
+      ENV['FQDN'] = expected_fqdn
+    end
```

Comment:
I think this may need a variable to capture current FQDN and a paired `after` block (or an `around`)  to set it back. Else we have only moved the sneaky global to another place.